### PR TITLE
XLS-0033d-CFTs: Make AssetScale optional

### PR DESCRIPTION
**Pull request by sappenin**
Per this [PR comment](https://github.com/ledhed2222/rippled/pull/1/files#r1255074771), this change makes `AssetScale` optionally supplied in `CFTokenIssuanceCreate` transactions.

### CODE REVIEW

**Review by intelliot — APPROVED** [review]
I think this is fine, but why is it called `AssetScale`? Does it have the same purpose/meaning as [ERC-20](https://eips.ethereum.org/EIPS/eip-20)'s `decimals`? If so, I would prefer to call it `Decimals` for better ease of understanding.

**Comment by sappenin** [comment]
> I think this is fine, but why is it called `AssetScale`? Does it have the same purpose/meaning as [ERC-20](https://eips.ethereum.org/EIPS/eip-20)'s `decimals`? 

Yes, I think the two meanings are the same. What do you think about also using `Symbol` instead of `AssetCode`?

**Comment by intelliot** [comment]
> Yes, I think the two meanings are the same. What do you think about also using `Symbol` instead of `AssetCode`?

Sounds good to me!

**Comment by sappenin** [comment]
> https://github.com/XRPLF/XRPL-Standards/pull/127#issuecomment-1680921275

@ledhed2222 @nkramer44 @seelabs Any opinion on this terminology change?

File: XLS-0033d-CFTs/README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Affiliation: <a href="https://ripple.com">Ripple</a>
 
 # 1. Compact Fungible Tokens (CFTs)
 
-## 1.1. Abstract
+## 1.1. Abstract
```
Comment (ledhed2222) [inline XLS-0033d-CFTs/README.md:14]: errant space

**Review by ledhed2222 — APPROVED** [review]

**Comment by ledhed2222** [comment]
actually - i've incorporated this into my other PR https://github.com/XRPLF/XRPL-Standards/pull/123

since that hasn't been reviewed yet, let's make it a place for ongoing changes rather than having multiple open PRs in this repo

**Review by ledhed2222 — CHANGES_REQUESTED** [review]
see note - please close :)

**Comment by sappenin** [comment]
> actually - i've incorporated this into my other PR https://github.com/XRPLF/XRPL-Standards/pull/123 Since that hasn't been reviewed yet, let's make it a place for ongoing changes rather than having multiple open PRs in this repo

Sounds great, closing this PR in favor of https://github.com/XRPLF/XRPL-Standards/pull/123

**Comment by sappenin** [comment]
@ledhed2222 - given that there's a lot of discussion in your [new CFT spec PR](https://github.com/XRPLF/XRPL-Standards/pull/123), how do you feel about me re-opening this one so we don't have a bunch of open issues being debated in one place (i.e., your other PR)?

(Trying to reduce as much noise as possible).

**Comment by sappenin** [comment]
Closing this PR in-favor of https://github.com/XRPLF/XRPL-Standards/pull/137, which has everything in this PR.

**Pull request closed without merging**